core/src/main/java/com/datastax/oss/driver/internal/core/control/ControlConnection.java

```diff
@@ -399,39 +415,44 @@ private void connect(
                       }
                     } else if (closeWasCalled || initFuture.isCancelled()) {
                       LOG.debug(
-                          "[{}] New channel opened ({}) but the control connection was closed, closing it",
+                          "[{}] New channel opened ({}) to {} but the control connection was closed, closing it",
                           logPrefix,
-                          channel);
+                          channel,
+                          node);
                       channel.forceClose();
                       onSuccess.run();
                     } else if (lastDistanceEvent != null
                         && lastDistanceEvent.distance == NodeDistance.IGNORED) {
                       LOG.debug(
-                          "[{}] New channel opened ({}) but node became ignored, "
+                          "[{}] New channel opened ({}) to {} but node became ignored, "
                               + "closing and trying next node",
                           logPrefix,
-                          channel);
+                          channel,
+                          node);
                       channel.forceClose();
                       connect(nodes, errors, onSuccess, onFailure);
                     } else if (lastStateEvent != null
                         && (lastStateEvent.newState == null /*(removed)*/
                             || lastStateEvent.newState == NodeState.FORCED_DOWN)) {
                       LOG.debug(
-                          "[{}] New channel opened ({}) but node was removed or forced down, "
+                          "[{}] New channel opened ({}) to {} but node was removed or forced down, "
                               + "closing and trying next node",
                           logPrefix,
-                          channel);
+                          channel,
+                          node);
                       channel.forceClose();
                       connect(nodes, errors, onSuccess, onFailure);
                     } else {
-                      LOG.debug("[{}] New channel opened {}", logPrefix, channel);
+                      LOG.debug("[{}] New channel opened {} to {}", logPrefix, channel, node);
                       DriverChannel previousChannel = ControlConnection.this.channel;
                       ControlConnection.this.channel = channel;
                       if (previousChannel != null) {
                         // We were reconnecting: make sure previous channel gets closed (it may
                         // still be open if reconnection was forced)
                         LOG.debug(
-                            "[{}] Forcefully closing previous channel {}", logPrefix, channel);
+                            "[{}] Forcefully closing previous channel {}",
+                            logPrefix,
+                            previousChannel);
```

Comment:
Note: I think this was supposed to be `previousChannel` so fixing that at the same time